components/table/Table.tsx

```diff
@@ -96,6 +97,7 @@ export interface TableProps<RecordType>
   };
   sortDirections?: SortOrder[];
   showSorterTooltip?: boolean;
```

Comment:
```jsx
showSorterTooltip={{ placement: 'top' }}
```

Reply:
https://github.com/ant-design/ant-design/blob/master/components/slider/index.tsx#L46

Reply:
I believe `showSorterTooltip={{ ... }}` is a better API.